src/models/event-leaderboard.js

```diff
@@ -0,0 +1,25 @@
+/* eslint no-underscore-dangle: 0 */
+const { isFinite, isString } = require('lodash');
+const { toLowerCase } = require('../utils');
+
+module.exports = class GlobalLeaderboard {
+  constructor(params) {
+    this.validate(params);
+    this.format(params);
+  }
+
+  validate(params) {
+    if (!isString(params.userAddress)) throw Error('userAddress must be a String');
+    if (!isString(params.eventAddress)) throw Error('eventAddress must be a String');
+    if (!isString(params.investments)) throw Error('investments must be a Number');
+    if (!isFinite(params.winnings)) throw Error('winnings must be a Number');
+  }
+
+  format(params) {
+    this.eventAddress = toLowerCase(params.eventAddress);
+    this.userAddress = toLowerCase(params.userAddress);
+    this.investments = params.investments;
+    this.winnings = params.winnings;
+    this.returnRatio = this.winnings / this.investments;
```

Comment:
cant do this with two satoshi strings so you need to convert to BN. but there is a caveat, you cant just divide it as BNs since it does not handle floating point precision.

so Web3 seems like they are considering BigNumber.js as a temp for handling this, but the reason they took it out was for performance issues (https://github.com/ethereum/web3.js/issues/2171).

I guess for our case, we can use BigNumber for this specific case to handle the precision and percentages.